test/e2e/utils/github.ts

```diff
@@ -0,0 +1,49 @@
+import { Page } from "@playwright/test";
+import { GithubTestDataRoles, testData } from "test/e2e/constants";
+import { e2eEnvVars } from "test/e2e/env-e2e";
+
+const data = testData.github;
+export const githubLogin = async (page: Page, roleName: keyof GithubTestDataRoles, saveState = false) => {
```

Comment:
This looks like it is never used? it's commented out in setup.ts, is that just commented while you solve it or didn't work?

Reply:
it works, but it gets blocked and asks for 2FA.  The next PR will have that in, this is just the start :)